src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -116,6 +118,7 @@ class AddressValidationModal extends React.Component {
     } = address;
 
     const isAddressFromUser = id === 'userEntered';
+    const isNotSingleSuggestion = confirmedSuggestions.length > 1;
```

Comment:
`greaterThanOneSuggestion`? This is a hard variable to come up with a good variable name haha